File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program23.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		a program which accept distance in kilometre and convert it into meter. (1kilometre = 1000 Meter)
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{
	public int iValue;

	public Solution(int iValue)
	{
		this.iValue = iValue;
	}

	public int KMtoMeter()
	{

                return iValue*1000;
	
	}
}

class Program3
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);

		int iRet=0;

		System.out.println("Enter Distance: ");
		int iValue = sObj.nextInt();

		Solution obj = new Solution(iValue);
		iRet = obj.KMtoMeter();
	
		System.out.println("KM to meter conversion is: "+iRet);
	}
}

/*
OUTPUT:

Enter Distance: 
5
KM to meter conversion is: 5000
`
*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program23.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		a program which accept distance in kilometre and convert it into meter. (1kilometre = 1000 Meter)
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class Ans
{
	private:
		int iNo;
		int iConvert;

	public:
		Ans(int iNo,int iConvert=0)
		{
			this->iNo = iNo;
		}

		double KMtoMeter()
		{
			iConvert = iNo*1000;
			return iConvert;
		}
};		

int main()
{
	int iValue = 0,iRet = 0;

	cout<<"Enter distance";
	cin>>iValue;

	Ans *aObj = new Ans(iValue);
	iRet = aObj->KMtoMeter();

	cout<<"KM to meter conversion is : "<<iRet<<"\n";

	return 0;
}

/*

	OUTPUT:

		Enter distance5
		KM to meter conversion is : 5000

		Enter distance12
		KM to meter conversion is : 12000

*/
